uwblueprint/project-read-frontend
# Invite a user dialog

### Notion ticket link



[Invite a user dialog](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=74f12651da20462cbc5dad211020e397)



### Implementation description

- Created an "invite a user" dialog that calls the POST /users endpoint
<img width="953" alt="Screen Shot 2021-09-09 at 12 42 34 AM" src="https://user-images.githubusercontent.com/47638847/132623798-a88e8b31-693a-4250-bec4-3bd00ce3bada.png">



### Steps to test

1. Run both frontend and backend apps. Go to http://localhost:3000/users and try adding a new user. Make sure that the user shows up in backend as well.



### What should reviewers focus on?

- Clean code, good styling
- Have a good day 🌈 

### Checklist
